docs/winshell/INSTALL.md

```diff
@@ -0,0 +1,32 @@
+# OpenKH Documentation - WinShell Extensions
+In order to allow many quality of life improvements to aid in modding Kingdom Hearts games, OpenKH features several WinShell extensions to serve many different purposes.
```

Comment:
Sorry to be a pain, bu tbetween the header and the paragraph there must be a new line. I suggest to use Visual Studio Code to know where the warnings are.

Reply:
I see. I made this in Stackedit.IO because of it's preview function, will fix.